website/docs/r/vmware_cluster.html.markdown

```diff
@@ -0,0 +1,90 @@
+---
+subcategory: "VMware (AVS)"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_vmware_cluster"
+description: |-
+  Manages a Vmware Cluster.
+---
+
+# azurerm_vmware_cluster
+
+Manages a Vmware Cluster.
+
+## Example Usage
+
+~> **NOTE :**  Normal `terraform apply` could ignore this note. Please disable correlation request id for continuous operations in one build (like acctest) in Vmware. The continuous operations like `update` or `delete` could not be triggered when it shares the same `correlation-id` with its previous operation in Vmware.
+
+```hcl
+provider "azurerm" {
+  features {}
+  disable_correlation_request_id = true
+  # In Vmware Acctest (or continuous operations in one build), we need to disable correlation request id to prevent following operations are not triggered 
+}
```

Comment:
```suggestion
~> **NOTE :**  Normal `terraform apply` could ignore this note. Please disable correlation request id for continuous operations in one build (like acctest) in Vmware. The continuous operations like `update` or `delete` could not be triggered when it shares the same `correlation-id` with its previous operation in Vmware.

```hcl
provider "azurerm" {
  features {}
  disable_correlation_request_id = true
  # In Vmware Acctest (or continuous operations in one build), we need to disable correlation request id to prevent following operations are not triggered 
}
```

wait, so this is only a problem in acctests? not with normal tf plan and apply? if soe we should remove this from the docs and not say anything. normal users need to know about this - just leave a comment in the tests explaining it and linking to an issue on the SDK/something tracking the bug